File: abc012/src/c.cpp
#include <iostream>

using namespace std;

int main()
{
    int n;
    cin >> n;
    int all = 0;
    for (int a = 1; a <= 9; a++)
    {
        for (int b = 1; b <= 9; b++)
        {
            all += a * b;
        }
    }
    int x = all - n; // xが足りない数字
    for (int a = 1; a <= 9; a++)
    {
        for (int b = 1; b <= 9; b++)
        {
            if (a * b == x)
            {
                cout << a << " x " << b << endl;
            }
        }
    }
    return 0;
}

File: abc012/src/c.go
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)
	all := 0
	for a := 1; a <= 9; a++ {
		for b := 1; b <= 9; b++ {
			all += a * b
		}
	}
	x := all - n
	for a := 1; a <= 9; a++ {
		for b := 1; b <= 9; b++ {
			if a*b == x {
				fmt.Printf("%d x %d\n", a, b)
			}
		}
	}
}
